install/kubernetes/helm/istio/charts/mixer/templates/istio-mixer.yaml

```diff
@@ -190,6 +190,41 @@ spec:
     singular: listchecker
   scope: Namespaced
   version: v1alpha2
+
+---
+
+kind: CustomResourceDefinition
+apiVersion: apiextensions.k8s.io/v1beta1
+metadata:
+  name: kuberneteses.config.istio.io
+  labels:
+    package: adapter.template.kubernetes
+    istio: mixer-instance
+spec:
+  group: config.istio.io
+  names:
+    kind: kubernetes
```

Comment:
i wonder if this will ever be confusing for operators. I don't have any great suggestions, however.